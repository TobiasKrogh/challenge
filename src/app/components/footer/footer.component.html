<a routerLink="/tac">Terms&Conditions | Privacy policy</a>
