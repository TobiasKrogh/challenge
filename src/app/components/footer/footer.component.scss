:host {
  align-items: center;
  display: flex;
  height: 3.75rem;
  padding-left: var(--navigation-width);
}
