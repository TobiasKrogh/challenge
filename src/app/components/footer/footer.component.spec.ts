import { ComponentFixture, TestBed } from '@angular/core/testing';
import { RouterTestingModule } from '@angular/router/testing';

import { FooterComponent } from './footer.component';

describe('FooterComponent', () => {
  let component: FooterComponent;
  let fixture: ComponentFixture<FooterComponent>;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      imports: [RouterTestingModule],
      declarations: [FooterComponent],
    }).compileComponents();

    fixture = TestBed.createComponent(FooterComponent);
    component = fixture.componentInstance;
  });

  it('should create', () => {
    fixture.detectChanges();
    expect(component).toBeTruthy();
  });

  describe('When rendering', () => {
    it('Then it matches snapshot', () => {
      fixture.detectChanges();
      expect(fixture.debugElement).toMatchSnapshot();
    });
  });
});
