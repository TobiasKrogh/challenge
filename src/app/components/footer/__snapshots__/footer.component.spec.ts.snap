// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterComponent When rendering Then it matches snapshot 1`] = `
DebugElement {
  "nativeNode": <div
    id="root1"
  >
    <a
      href="/tac"
      routerlink="/tac"
    >
      Terms&Conditions | Privacy policy
    </a>
  </div>,
}
`;
